### ISSUE TITLE
Latest APK

### ISSUE THREAD

**Issue by MobCode100** [renyuneyun/Easer#480]
This is not a bug report, just to request a latest beta or at least, provide a [v0.8.3-beta3](https://github.com/renyuneyun/Easer/releases/tag/v0.8.3-beta3) APK file so that it's easier for the community to test them and report issues. There are some critical fix (such as notification event) and features which have been made into beta/dev branch that aren't included in the latest and beta APK releases.

However, I've build my own a [latest Easer APK](https://github.com/MobCode100/Easer/releases/tag/v0.8.3-beta3-edge) for anyone who wants to use it.

**Comment by svolpe** [comment]
@MobCode100 I'm having issues getting Easer to even build. I have tons of programming experience but I'm fairly new to Android. Would you be able to send me some directions on how to compile it from scratch. I'm getting the following error:

Unsupported Java. 
Your build is currently configured to use Java 17.0.9 and Gradle 6.5.

I went down the path of upgrading the product to use a newer version of Gradle but then I started getting other issues so I'm assuming that might not have been the right approach.

**Comment by svolpe** [comment]
If I change the Java version to one that supports Gradle 6.5 then I get the following error:
Could not resolve all dependencies for configuration ':app:betaDebugRuntimeClasspath'.
Could not create task ':app:minifyBetaReleaseWithR8'.
Cannot query the value of this provider because it has no value available.


I think I got the same error if I updated the Gradle instead of downgrading  Java.

**Comment by MobCode100** [comment]
I'd get that error too, so I solved it with:
Go to File > Project Structure > Modules > app > Properties
![image](https://github.com/renyuneyun/Easer/assets/66469454/8f2c946a-813c-42d4-8a81-82a17d308336)

Change the compile sdk version and build tools version to 30.

Make sure you download the build tools:
![image](https://github.com/renyuneyun/Easer/assets/66469454/7b5f45a4-48dd-49ae-aa1e-9190ac306cae)

Tick `Hide Obsolete Packages`.

**Comment by svolpe** [comment]
did you leave Gradle as the original version and install an old version of JDK or did you upgrade it too?

**Comment by MobCode100** [comment]
> did you leave Gradle as the original version and install an old version of JDK or did you upgrade it too?

No, I just leave everything as it is. To be honest I'm not familiar with android development too. I made just those changes, and then they worked for me.

**Comment by svolpe** [comment]
yeah, they worked for me too. Thank you for your help!

**Comment by MobCode100** [comment]
> yeah, they worked for me too. Thank you for your help!

Glad it worked for you. I've tried to upgrade it using android studio recommendation but it failed to compile. So I'll just stick to SDK 30.
